app/src/main/java/info/nightscout/android/model/medtronicNg/StatusEvent.java

```diff
@@ -0,0 +1,301 @@
+package info.nightscout.android.model.medtronicNg;
```

Comment:
Looking further down, it looks like it's being used for more than just testing. I'm wondering if there's a more manageable way to do this without duplication of code and memory in the runtime...